### PR TITLE
Update deployment target to iOS 13

### PR DESCRIPTION
**Pull request by sujaypat**

### CODE REVIEW

File: HackIllinois/ViewControllers/HIBaseViewController.swift [hunk 1]
```diff
@@ -255,21 +255,23 @@ extension HIBaseViewController {
         NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillHideNotification, object: nil)
     }
 
+    /// Needs to updated for iOS 13.0. Not currently used anywhere.
```
Comment (sujaypat) [inline HackIllinois/ViewControllers/HIBaseViewController.swift]: this should be a TODO

**Comment by PatrickKan** [comment]
Might reconsider updating to iOS 13 based on feedback

**Comment by PatrickKan** [comment]
Might want to reconsider keeping iOS 11 to make the app more compatible and reduce the amount of people updating their iOS while in line to the event

**Comment by lasyaneti** [comment]
Outdated PR

**Pull request closed without merging**